03Lesson/01Project/css/style.css

```diff
@@ -1 +1,51 @@
 
+body {
+  font-family: 'Arial', sans-serif;
+}
+
+header, main, footer {
+  width: 1024px;
+  font-family: sans-serif;
+  margin: auto;
+}
+
+header {
+  background-color: #0071c5;
+  padding: 10px;
+  color: white
+}
+
+header-#h1 {

```

Comment:
Interesting selector, is that `-` there by accident? 😄 Doesn't look like valid CSS